overlord/servicestate/servicestatetest/quotas.go

```diff
@@ -30,3 +34,24 @@ import (
 func PatchQuotas(st *state.State, grps ...*quota.Group) (map[string]*quota.Group, error) {
 	return internal.PatchQuotas(st, grps...)
 }
+
+func MockQuotaInState(st *state.State, quotaName string, parentName string, snaps []string, memoryLimit quantity.Size) error {
+	osutil.MustBeTestBinary("cannot mock quota group in state outside of tests")
```

Comment:
as this is a *test package we don't strictly need this

Reply:
since this PR is a skip spread one, I will push this change now

Reply:
done